include/micm/solver/rosenbrock.hpp

```diff
@@ -76,7 +76,7 @@ namespace micm
     // Gamma_i = \sum_j  gamma_{i,j}
     std::array<double, 6> gamma_{};
 
-    double absolute_tolerance_{ 1e-12 };
+    double absolute_tolerance_{ 1e-3 };
```

Comment:
I'm going to create a separate issue to figure out how to handle absolute tolerances. They will depend on the units, which for just the gas-phase we can maybe base them on our default units of `mol m-3`, but if we add aerosol-phase species, we might have to rethink this.